sql/004_search.sql

```diff
@@ -845,7 +843,7 @@ context := jsonb_strip_nulls(jsonb_build_object(
 
 collection := jsonb_build_object(
     'type', 'FeatureCollection',
-    'features', out_records,
+    'features', coalesce(out_records, '[]'::jsonb),
```

Comment:
simple fix to make sure `features` is at least set to `[]`.

I couldn't track down where in https://github.com/stac-utils/pgstac/pull/46/files#diff-3c3e93de12c078e7f46e264188fcc2ba82eefd77be73795f964ad132d89af285R835 the transformation from `[]` to `null` was 🤷

Reply:
😬 tests still fails 🤷‍♂️ 
😭 but it works locally 😭